src/main.rs

```diff
@@ -212,6 +212,8 @@ fn do_main() -> Result<i32> {
         RevisionOpt::Revision(revision)
     } else if let Some(merge_base_with) = args.merge_base_with {
         RevisionOpt::MergeBaseWith(merge_base_with)
+    } else if !lint_runner_config.merge_base_with.is_empty() {
+        RevisionOpt::MergeBaseWith(lint_runner_config.merge_base_with.clone())
```

Comment:
If merge_base_with is set, there is no longer a way to run on commits only. That seems bad?

Reply:
You mean no way to run it on just the last commit only?

Folks could still do it by passing `--revision HEAD~` right?

Reply:
okay sg